packages/react/package.json

```diff
@@ -134,7 +134,7 @@
       "name": "Geo",
       "path": "dist/esm/index.js",
       "import": "{ MapView, LocationSearch }",
-      "limit": "300 kB"
+      "limit": "330 kB"
```

Comment:
I'm now sure we're OK with bumping this limit given this is a jump from 280kB to 330kB, or about an 11.5% jump for an app bundle using `MapView` and `LocationSearch`

Reply:
How big is the bundle now? Exactly 330k?

Reply:
It's now:
321.52 kBwith all dependencies, minified and gzipped